### PR TITLE
Package

### PR DESCRIPTION
**Pull request by vascotenner**
Created a package that can be installed via PIP

### CODE REVIEW

**Comment by vascotenner** [comment]
Tested this in both python2 and python3.
Replaced - in filenames to be complaint with pep8

The setup creates also two scripts that are placed in a PATH, and hence do not need a ./ anymore:
1. ```find_port```
2. ```usb_ser_mon```

File: find_port.py [hunk 1]
```diff
@@ -127,25 +124,16 @@ def main():
...
-        detected = False
-        for device in context.list_devices(subsystem='tty'):
-            if is_usb_serial(device, args):
-                print('USB Serial Device %s:%s%s found @%s\r' % (
-                      device['ID_VENDOR_ID'], device['ID_MODEL_ID'],
-                      extra_info(device), device.device_node))
-                detected = True
-        if not detected:
-            print('No USB Serial devices detected.\r')
-        return
```
Comment (dhylands) [inline find_port.py]: This return shouldn't have been removed. The way things are currently coded it does both the --list functionality and then the non --list functionality.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    #
+    # This field corresponds to the "Description-Content-Type" metadata field:
+    # https://packaging.python.org/specifications/core-metadata/#description-content-type-optional
+    long_description_content_type='text/markdown',  # Optional (see note above)
+
+    # This should be a valid link to your project's main homepage.
+    #
+    # This field corresponds to the "Home-Page" metadata field:
+    # https://packaging.python.org/specifications/core-metadata/#home-page-optional
+    url='https://github.com/vascotenner/usb-ser-mon/',  # Optional
```
Comment (dhylands) [inline setup.py]: Shouldn't this point to my repository?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+    # This should be your name or the name of the organization which owns the
+    # project.
+    author='',  # Optional
```
Comment (dhylands) [inline setup.py]: You can put my name here and my email dhylands@gmail.com.

I consider the effort to do the packaging to be significant and you can also add your name/email, if desired.

**Pull request merged**